src/main/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package main.service.impl;
+
+import main.service.interfaces.FileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    private static final String READ_FAILED = "Can't read file ";
```

Comment:
this is redundant. It's ok to write exception message directly when instantiate an exception